docs/topic-guides/blendable-web-apps/common-look-and-feel.md

```diff
@@ -0,0 +1,84 @@
+
+### Common UI patterns and components of Uniform design system
+
+As the basic way for providing a common look and feel in the presentation layer, Starcounter provides Uniform design system. It consists of custom CSS properties, CSS reset and a library of patterns, and components.
+
+Uniform follows the inversion of control principle, which makes it a link between app authors and solution owners.
+
+Unlike many design systems, Uniform is open-ended and themeable. It can be configured, extended or replaced in parts to achieve anything that is possible with HTML, CSS, and JavaScript.
+
+The custom CSS properties are the basic set of primitives. They define the fonts, sizes, colors, margins and other values used throughout the reset, patterns and components.
+
+The CSS reset overwrites the default user agent stylesheet. It provides the look of the native HTML5 elements such as the headings (H1-H6), links, inputs, buttons, and tables. It is implicitly loaded by the Starcounter app shell, so there is no need to load it explicitly in the views. 
+
+The pattern library, [Uniform.css](https://starcounter.github.io/uniform.css/), is a set of CSS classes that implement visual hierarchy and give specific roles to native elements. It provides sections, cards, titles, column layouts, alerts,  buttons, etc.
+
+The component library, [Uniform components](https://starcounter.github.io/uniform.css/components/), is a set of custom elements that implement interactive UI controls. For the beginning, we provide a form item, a date picker, a data table and a pagination control. 
+
+To use the patterns and the components, you must import them explicitly. They are placed in the shadow root of the view container to wrap the distributed content elements. That allows to serve real progressive enhancement, and what is crucial for a solution of blended apps: keep app content's markup semantic and unopinionated, giving solution owner a way to apply own opinion without changing the app source code.
+
+The names of the custom CSS properties, CSS classes, and custom elements begin with a `uni-` prefix.
+
+Our [UniformDocs](https://uniform.starcounter.io/) app demonstrates the use of Uniform design system as well as commonly used 3rd party custom elements that we can endorse, such as a Google map or a combo box.
+
+As an app author, you are invited to implement specialized components in your app that use Uniform internally. This will allow to adapt to the solution’s look and feel also for your specialized components.
+
+### Theme configuration through custom CSS properties
+
+Uniform design system comes with a modern-looking default theme.
+
+As a solution owner, you can customize it by providing new values for the [custom CSS properties](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_variables). This allows changing the fonts, colors, etc. without replacing the stylesheet.
```

Comment:
We're talking about custom CSS properties earlier, but this is the first time we link to it. Only the first mention should be a link. 

In general, avoid using "etc." Both the [Microsoft](https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/etc) and [Google](https://developers.google.com/style/word-list#letter-E) technical writing style guides advice against it.